python/lsst/daf/butler/configs/storageClasses.yaml

```diff
@@ -284,3 +284,5 @@ storageClasses:
     pytype: spectractor.extractor.spectrum.Spectrum
   SpectractorImage:
     pytype: spectractor.extractor.images.Image
+  SpectractorPickle:
```

Comment:
I'd rather we avoid defining a storage class name after it's serialization format if at all possible. The storage class should hew closer to the python type rather than the output file format.

Reply:
Yeah, that was a deliberate choice to be honest, because it's so fragile and shouldn't be around to stay, so wanted people using it to be aware that they're touching a pickle. I can remove it if you prefer, but this was a conscious choice to try to mark it as dodgy.